packages/create-sitecore-jss/src/templates/node-headless-ssr-proxy/httpAgents.js

```diff
@@ -0,0 +1,42 @@
+const keepAlive = require("agentkeepalive");
+const http = require("http");
+const https = require("https");
+
+const keepAliveConfig = {
```

Comment:
Something happened with the indents in this file, can you, please, run prettier? I know that it was initially, easy to fix while we are working on it

Reply:
Fixed